src/packages/excalidraw/README_NEXT.md

```diff
@@ -470,6 +471,10 @@ const excalidrawRef = { current: { readyPromise: <a href="https://github.com/exc
 
 Since plain object is passed as a `ref`, the `readyPromise` is resolved as soon as the component is mounted. Most of the time you will not need this unless you have a specific use case where you can't pass the `ref` in the react way and want to do some action on the host when this promise resolves. You can check the [example](https://codesandbox.io/s/eexcalidraw-resolvable-promise-d0qg3?file=/src/App.js) for the usage.
 
+#### `app`
```

Comment:
Missed your comment. Initially this API was to solve a case with the image API. I did make a [change later](https://github.com/excalidraw/excalidraw/pull/4089) which obviated this. But, there will be cases like this where either we don't have an API yet, it works differently, or we won't expose an API for this.

Furthermore, I think it's better to expose the app for cases like getting access to the `scene`/`manager` instances etc. instead of creating and exposing getters and setters for everything.

Is there an argument against this?